packages/web-components/fast-ssr/src/request-storage.ts

```diff
@@ -2,7 +2,7 @@ import { AsyncLocalStorage } from "async_hooks";
 import { DI, DOMContainer } from "@microsoft/fast-element/di";
 import { createWindow } from "./dom-shim.js";
 
-const asyncLocalStorage = new AsyncLocalStorage();
+let asyncLocalStorage = new AsyncLocalStorage();
```

Comment:
In the case where module is loaded with `require-from-string`, doing `import { RequestStorageManager} from "@microsoft/fast-ssr"` do not benefit from module cache and will recreate a new AsyncLocalStorage object at import time even though it was meant to be replaced with RequestStorageManager.set(storage) for each request. Is this a perf hit concern? Or would it be better pattern to allow lazy construction of asyncLocalStorage?

Reply:
I wouldn't be worried about the construction of this object. From my research, perf doesn't come into play at this stage in the use of AsyncLocalStorage.